src/robot/libraries/Collections.py

```diff
@@ -346,7 +346,7 @@ def list_should_not_contain_duplicates(self, list_, msg=None):
                                  '%s found multiple times.' % seq2str(dupes))
 
     def lists_should_be_equal(self, list1, list2, msg=None, values=True,
-                              names=None):
+                              names=None,ignore_order=False):
```

Comment:
missing space between comma and ignore_order

Reply:
Resolved. learning many things in 2nd PR. thanks.